### PR TITLE
QR Code scanning documentation

### PR DESCRIPTION
**Pull request by kikettas**
Added QR Code scanning guide to documentation

### How to test
Nothing to test.

### Merging
Automatic.

### CODE REVIEW

File: .jazzy.json [hunk 1]
```diff
@@ -21,10 +21,11 @@
       {
          "name":"Documentation",
          "children":[
-	    "Installation",
-	    "Updating to 3.2.0",
+	          "Installation",
```
Comment (a-szotyori) [inline .jazzy.json]: What happened to these indentations?

Comment (kikettas) [reply .jazzy.json]: Honestly I didn't use a proper editor when I created it for the first time, so I had to fix the indentation

File: Documentation/source/QR Code scanning guide.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+QR Code Scanning
+=============================
+
+Some invoices have a QR Code that allows the user to get the payment data just scanning it from the camera screen. If the QR Code has a valid format (see [supported QR Codes](#supported-qr-codes)), a popup appears pointing out that a QR Code has been detected and allowing the user to use it.
+<center><img src="img/qr_code_popup.jpg" border="1"/></center>
```
Comment (nsobadjiev) [inline Documentation/source/QR Code scanning guide.md]: Isn't the image a bit smudged?

File: Documentation/source/QR Code scanning guide.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+QR Code Scanning
+=============================
+
+Some invoices have a QR Code that allows the user to get the payment data just scanning it from the camera screen. If the QR Code has a valid format (see [supported QR Codes](#supported-qr-codes)), a popup appears pointing out that a QR Code has been detected and allowing the user to use it.
```
Comment (nsobadjiev) [inline Documentation/source/QR Code scanning guide.md]: I'd prefer "Just *by* scanning it..."

Comment (nsobadjiev) [inline Documentation/source/QR Code scanning guide.md]: Why is Code in QR Code always capitalised? It's not really a name.

File: Documentation/source/QR Code scanning guide.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+...
+...
+...		
+giniConfiguration.qrCodeScanningEnabled = true
+```
+
+Handle the Payment Data
+----------------------
+
+Once the QR Code has been detected and the user has tapped the button to use it, the payment data is returned. In order to handle the Payment Data from the QR Code, on the one hand if you are using the _Screen API_ you have to implement the delegate method `GiniVisionDelegate.didDetect(qrDocument:)` to get the `GiniQRCodeDocument`.
```
Comment (nsobadjiev) [inline Documentation/source/QR Code scanning guide.md]: "On one hand" ... "on the other hand". The first hand is not *the* hand :)

File: Documentation/source/QR Code scanning guide.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+```
+
+Customization
+----------------------
+It is possible to customize the text label, button and background colors with these parameters:
+- `GiniConfiguration.qrCodePopupBackgroundColor`
+- `GiniConfiguration.qrCodePopupButtonColor`
+- `GiniConfiguration.qrCodePopupTextColor`
+
+Also the text from both label and button can be customized through the following parameters in your `Localizable.strings` file:
```
Comment (nsobadjiev) [inline Documentation/source/QR Code scanning guide.md]: "*Additionally* the text from both *the* label and *the* button" would sound better IMO.

File: Documentation/source/QR Code scanning guide.md (hunk 2 above)
Comment (kikettas) [reply Documentation/source/QR Code scanning guide.md]: No, it is ok!

**Pull request merged**